src/css/consultancy-page.css

```diff
@@ -0,0 +1,121 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+   }
+  /* To reset some default browser styles */
+   body, p , ul, li {
+    margin: 0 ;
+    padding: 0 ;
+   }
+  
+    /* To Apply a background color to the body */
+   body {
+    background-color: #d4c5ee;
+    font-family: Arial, sans-serif;
+  }
+  
+    /* To style the header */
+   header {
+    color: #000000;
+    text-align:centre;
+    padding: 20px 0;
+    font-size: 24px;
+   }
+    
+   /* To style the main content container */
+   .container{
+    max-width: 500px;
+    margin: auto;
+    padding: 30px;
+   }
+  
+  
+   /* To style the about section */
+   .about {
+    background-color: #ffffff;
+    padding: 20px;
+    border-radius: 5px;
+    box-shadow:  5px rgba(0,0,0,0)
+   }
+  /* To select the boxes */
+  select{
+    color:black;
+    width: 10%;
+    padding: 10px;
+    margin-top: 10px;
+    height: auto;
+    margin-inline-start: 30px;
+    border: 1px solid #000000;
+  
+    }
+   
+  /* To Style the button */
+  button{
+    background-color: #231f54;
+    color: #fff;
+    border: none;
+    padding: 15px;
+    margin-left: 20px;
+    border-radius: 15px;
+    cursor:pointer;
+    line-height: 5px;
+    size-adjust: 250px;
+   }
+  
+   /* To add spacing between elements */
+  br{
+    margin-bottom:30px;
+  }
+  
+  /* To style the image */
+  img{
+    width: 20%;
+    height: 70px;
+    margin-inline-start: 20px;
+    margin-top: 15px;
+    margin-bottom: 0px;
```

Comment:
use a single margi  property i.e margin:34px 34px 34px 45px;  this here specifies the margin for top, right, bottom and left in a single line.